projects/text-editor/src/modules/text-editor/text-editor.component.ts

```diff
@@ -96,18 +95,10 @@ let nextUniqueId = 0;
   selector: 'sky-text-editor',
   templateUrl: './text-editor.component.html',
   styleUrls: ['./text-editor.component.scss'],
-  providers: [
-    {
-      provide: NG_VALUE_ACCESSOR,
```

Comment:
☝️ Upgraded this provider to use the injected `NgControl` method so we could access the `control.setValue()` method (see constructor for implementation)
```
 this.ngControl.valueAccessor = this;
```